src/Pages/Login/Login.js

```diff
@@ -0,0 +1,169 @@
+    !email && !password ? setOnBlurHasValue(true) : setOnBlurHasValue(false);
+  };
+
+  const handleEmailCheck = (e) => {
+    e.preventDefault();
+
+    !/\S+@\S+\.\S+/.test(email)
+      ? setButtonsEmailHasValue(true)
+      : setButtonsEmailHasValue(false);
+
+    !/^.*(?=.{8,})(?=.*[0-9])(?=.*[a-zA-Z]).*$/.test(password)
+      ? setButtonsPwHasValue(true)
+      : setButtonsPwHasValue(false);
+  };
+
+  const handleKakaoCheck = () => {
+    alert("소셜 로그인 구현중...");
+  };
+
+  const { email, password } = inputs;
+  return (
+    <>
+      <Nav>
+        <H1>LOGO</H1>
+      </Nav>
+      <Main>
+        <Title>로그인</Title>
+        <Card>
+          <FormGroup>
+            <FormTitle>이메일</FormTitle>
+            <EmailPasswordInput
+              name="email"
+              type="email"
+              value={email}
+              invalidOnClick={buttonsEmailHasValue}
+              invalidOnBlur={onBlurHasValue}
+              onChange={handleUpdateValue}
+              onBlur={setOnBlurCheckValue}
+            />
+            {!email && buttonsEmailHasValue && (
+              <InvalidInfo>이메일 주소를 입력해주세요.</InvalidInfo>
+            )}
+            {email && buttonsEmailHasValue && (
+              <InvalidInfo>올바른 이메일 주소를 입력해주세요.</InvalidInfo>
+            )}
+          </FormGroup>
+          <FormTitle>비밀번호</FormTitle>
+          <EmailPasswordInput
+            name="password"
+            type="password"
+            value={password}
+            invalidPwOnClick={buttonsPwHasValue}
+            invalidOnBlur={onBlurHasValue}
+            onChange={handleUpdateValue}
+            onBlur={setOnBlurCheckValue}
+          />
+          {!password && buttonsPwHasValue && (
+            <InvalidInfo>비밀번호를 입력해주세요.</InvalidInfo>
+          )}
+          {password && buttonsPwHasValue && (
+            <InvalidInfo>영문+숫자 조합 8자리 이상 입력해주세요.</InvalidInfo>
+          )}
+          <StyledLink to="/findPassword">비밀번호 찾기</StyledLink>
+          <Buttons
+            emailClick={handleEmailCheck}
+            kakaoClick={handleKakaoCheck}
+          />
+          <StyledLink to="/signup">계정이 없으신가요?</StyledLink>
+        </Card>
+      </Main>
+    </>
+  );
+}
+
+export default Login;
+
+//logo 만들면 지우기!
+export const Nav = styled.nav`
+  background-color: #000;
+  text-align: center;
+  height: 72px;
+`;
+
+export const H1 = styled.h1`
+  color: #fff;
+  font-size: 24px;
+  font-weight: 600;
+`;
+
+export const Main = styled.main`
+  background-color: #fafafa;
+  padding: 5rem 0;
+`;
+
+export const Title = styled.h3`
+  text-align: center;
+  font-size: 34px;
+  font-weight: 600;
+  margin-bottom: 2.5rem;
+`;
+
+export const FormGroup = styled.div`
+  margin-bottom: 2rem;
+`;
+
+export const FormTitle = styled.p`
+  font-size: 1rem;
+  padding-bottom: 0.375rem;
+  font-weight: 500;
+`;
+
+export const InvalidInfo = styled.p`
+  display: ${(props) => {
+    if (!props.email) {
+      return "display";
+    }
+    return "none";
+  }};
```

Comment:
```suggestion
  display: ${({ email }) => email ? "none" : "display" };
```
가급적이면 함수 내에서 if 문 등으로 발생하는 중괄호는 생략해주시는게 좋을 것 같습니다.

Reply:
네 알겠습니다